LICENSE.txt

```diff
@@ -0,0 +1,671 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Since this isn't a core part of the platform, should it be Apache instead of AGPL?